src/factory/mod.rs

```diff
@@ -65,6 +65,9 @@ const QUAD: [Vertex; 4] = [
     },
 ];
 
+/// Mapping writer.
+pub type MapVertices<'a> = gfx::mapping::Writer<'a, BackendResources, Vertex>;
```

Comment:
IIRC, the only place we currently expose gfx-rs is custom pipelines, which to be fair could be hidden behind a feature flag just fine. You proposed a wrapper before, I think that was a good idea, totally fine for a follow-up.